resources/web/wwi/protoVisualizer/Node.js

```diff
@@ -311,9 +314,26 @@ export default class Node {
   };
 
   clearReferences() {
-    // TODO
+    this.def = new Map();
+
+    for (const [parameterName, parameter] of this.parameters)
```

Comment:
```suggestion
    for (const [, parameter] of this.parameters)
```
Is this working? that is what my linter is suggesting